### PR TITLE
Add signal and highlight colors to theme

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Add more colors to the theme

Update storybook

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1434** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1434" target="_blank">Graphite</a> 
    * **PR #1435** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1435" target="_blank">Graphite</a> 
      * **PR #1437** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1437" target="_blank">Graphite</a> 
        * **PR #1440** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1440" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1440?utm_source=stack-comment).

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1434** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1434" target="_blank">Graphite</a> 
    * **PR #1435** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1435" target="_blank">Graphite</a> 
      * **PR #1437** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1437" target="_blank">Graphite</a> 
        * **PR #1440** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1440" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1440?utm_source=stack-comment).

File: packages/ui/src/lib/theme/colors/colors.ts [hunk 1]
```diff
@@ -111,12 +141,32 @@ export const colors = {
...
-  lavender: purple[500],
+  lavender: oldPurple[500],
   textPrimary: gray[1000],
   textSecondary: gray[700],
   textTertiary: gray[500],
   textDisabled: gray[400],
   textNegative: gray[25],
+
+  // Signal & Highlight colors
+  blueFill1: highlight.blue.fill1,
```
Comment (alebedev) [inline packages/ui/src/lib/theme/colors/colors.ts:152]: Would be nice to comment when we should use fill1 vs fill2

**Review by alebedev — APPROVED** [review]
Looks good

File: packages/ui/src/lib/theme/colors/colors.ts (hunk 1 above)
Comment (gustaveen) [reply packages/ui/src/lib/theme/colors/colors.ts:152]: Right! fill1 will be used on light backgrounds and fill2 on dark backgrounds. Maybe a comment is enough or if we want it to reflect it in the name. Like `blueFillDark` & `blueFillLight` @robinandeer @alebedev

**Comment by robinandeer** [comment]
⏳ This pull request is set to merge as part of a Graphite merge job
Stack job ID: awvOMljvedZZqM3RSp8w.
See details on [graphite.dev](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/1440)

**Pull request merged**